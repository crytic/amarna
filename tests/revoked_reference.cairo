func revoked(res):
   let var_name = [ap]
   let asdf = 2
   bar(var_name)
   return (res=var_name)
end