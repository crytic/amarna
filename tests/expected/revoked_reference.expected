[dead-store] in revoked_reference.cairo:3:8
[revoked-reference] Potentially revoked reference in use because of a function call inbetween reference [declaration](0) and [usage](1). in revoked_reference.cairo:2:8 and revoked_reference.cairo:5:16
[unused-arguments] in revoked_reference.cairo:1:14
[unused-function] in revoked_reference.cairo:1:1